charts/lagoon-core/values.yaml

```
@@ -595,7 +595,7 @@ drushAlias:
 logsDBCurator:
   enabled: true
   image:
-    repository: amazeeiolagoon/logs-db-curator
+    repository: uselagoon/logs-db-curator
     pullPolicy: Always
     # Overrides the image tag whose default is the chart appVersion.
     tag: ""
```

Until https://github.com/amazeeio/lagoon/pull/2409 is released - we have to use the testlagoon version of this image
```suggestion
    repository: testlagoon/logs-db-curator
    pullPolicy: Always
    # Overrides the image tag whose default is the chart appVersion.
    tag: "pr-2409"
```